src/puma-cli.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+'use strict';
+const program = require('commander');
+const puma = require('pumascript');
+const fs = require("fs");
+const injector = require('./injector');
+
+
+
+let run = (req, op) => {
+    puma.evalPuma(fs.readFileSync(req, 'Utf8'));
+}
+
+
+let compile = (req, op) => {
+    if (req.split('.')[1] !== 'puma') {
```

Comment:
Be careful with extensions, file could contain dots in name. Best use something like `'file.js.puma'.split('.').reverse()[0]`

Reply:
I change it for:
```javascript
let compile = (req, op) => {
    let aux = req.split('.');   
    if (!aux.slice(1, aux.length).includes('puma')) {
        console.warn('The file entered does not have a .puma extension.');
```